crates/automerge-wasm/src/interop.rs

```diff
@@ -370,20 +370,23 @@ impl Automerge {
         datatype: Datatype,
         heads: Option<&Vec<ChangeHash>>,
         meta: &JsValue,
+        freeze: bool,
```

Comment:
If I understand correctly the purpose of this `freeze` argument is to track whether the object being patched (`nextDoc` in `progressDocument` should be `Object.freeze`ed after the patch is applied. Currently this only seems to be the case if the `Automerge` document was initialized with `freeze: true`, would it be possible then to just track the freeze state via a field on `automerge_wasm::Automerg` (e.g. `automerge_wasm::Automerge::freeze`)? That way we wouldn't need to feed the `freeze` argument through to every one of these functions.

Reply:
Ok - good call - I added an `enableFreeze()` method which I don't love but it shadows the `enabledPatches()` which is also a stand-in for a better API later.  We can fix it up at a later date.